demo/src/app/components/vapp-static-demo-component/vapp-static-demo.component.ts

```diff
@@ -0,0 +1,84 @@
+import { AfterViewInit, Component, ViewChild } from '@angular/core';
+import * as paper from 'paper';
+import { DemoComponent } from '../demo-component/demo.component';
+import { VappData, VappComponent } from '../../../../../src/components/vapp';
+import { placeholderArrayOfVappData } from '../../constants/vapp-basic-placeholder-data';
+import { ScrollbarComponent } from '../../../../../src/components/scrollbar';
+import { CANVAS_BACKGROUND_COLOR } from '../../../../../src/constants/colors';
+import { CONNECTOR_RADIUS, DEFAULT_SCROLLBAR_THICKNESS } from '../../../../../src/constants/dimensions';
+
+@Component({
+  selector: 'vapp-static-demo',
+  template: `
+	  <demo label="Static Design" height="730"
+          description="Demonstrates the static layout design of the vApp visual component for different cases."></demo>
+  ` })
+export class VappStaticDemoComponent implements AfterViewInit {
+
+  @ViewChild(DemoComponent)
+  demo: DemoComponent;
+
+  ngAfterViewInit() {
+    // sets up Paper Project
+    const proj = this.demo.getProject();
+    proj.activate();
+    const view = paper.view;
+    const canvas = paper.view.element;
+    this.demo.backgroundColor = CANVAS_BACKGROUND_COLOR;
+
+    const VIEW_PADDING = 30;
+    const DEMO_VAPP_TOP_ALIGNMENT = 59;
+    const VERTICAL_POSITION = VIEW_PADDING + DEMO_VAPP_TOP_ALIGNMENT + CONNECTOR_RADIUS;
+    const vapps: Array<VappData> = placeholderArrayOfVappData;
+
+    const content = new paper.Group({ applyMatrix: false });
+    // create origin paper Item for vapps to base position from
+    const origin = new paper.Path.Circle({
```

Comment:
VApps base their x position on the last child of the `content` group (line 44 - content.lastChild.bounds.right). Most vApps will have the previous vApp to refer to, but the very first one won't. The `origin` was created to be the first vApp's reference point. One way it could also be avoided is by doing a check in the loop at line 43:
```
vapps.forEach((vappData, index) => {
  const position = new paper.Point(index === 0 ? 'explicit number' : content.lastChild.bounds.right), VERTICAL_POSITION);
    ...
}
```